packages/taco/src/conditions/base/contract.ts

```diff
@@ -80,7 +81,7 @@ export const contractConditionSchema = rpcConditionSchema
     standardContractType: z.enum(['ERC20', 'ERC721']).optional(),
     method: z.string(),
     functionAbi: functionAbiSchema.optional(),
-    parameters: z.array(z.unknown()),
+    parameters: z.array(ParamOrContextParamSchema),
```

Comment:
Same comment here about `ParamSchema` not covering other valid options for parameters. See comment in `shared.ts`.

Some solidity functions can take lists (of various types) as parameters as well.